src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -0,0 +1,246 @@
+      setEmailInvalid(false)
+    }
+  }, [])
+
+  const handleOnSubscribe = useCallback(() => {
+    onSubscribe(emailAddress)
+  }, [emailAddress, onSubscribe])
+
+  return (
+    <div
+      css={`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        padding-top: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          text-align: center;
+        `}
+      >
+        <div
+          css={`
+            align-items: center;
+          `}
+        >
+          <img src={emailIllustration} width={141} height={141} alt="" />
+        </div>
+        <h3
+          css={`
+            ${textStyle('title2')};
+            margin-top: ${4 * GU}px;
+          `}
+        >
+          Verify your email address
+        </h3>
+
+        <span
+          css={`
+            ${textStyle('body2')};
+            color: ${theme.surfaceContentSecondary};
+            margin-top: ${1.5 * GU}px;
+          `}
+        >
+          Almost there! We’ve sent a verification email to {email}. Kindly check
+          your inbox and click the link to verify your account.
+          {updateMode &&
+            'Alternatively, you can update this email or delete it, if you wish to unsubscribe'}
+        </span>
+        {(() => {
+          if (!updateMode) {
+            return (
+              <Button
+                mode="strong"
+                onClick={onResendEmail}
+                css={`
+                  margin-top: ${3 * GU}px;
+                `}
+              >
+                Resend verification email
+              </Button>
+            )
+          }
+          return (
+            <>
+              <div
+                css={`
+                  margin-top: ${5 * GU}px;
+                `}
+              >
+                <Field label="Update email address">
+                  <TextInput
+                    value={emailAddress}
+                    adornment={
+                      emailInvalid ? (
+                        <IconCross
+                          css={`
+                            color: ${theme.negative};
+                          `}
+                        />
+                      ) : emailAddress.trim() ? (
+                        <IconCheck
+                          css={`
+                            color: ${theme.positive};
+                          `}
+                        />
+                      ) : (
+                        <IconCheck
+                          css={`
+                            opacity: 0;
+                          `}
+                        />
+                      )
+                    }
+                    adornmentPosition="end"
+                    type="email"
+                    wide
+                    onChange={handleEmailAddressChange}
+                    placeholder="email@address.com"
```

Comment:
This was mentioned elsewhere, but should we use `you@example.org`? Would like to make these consistent across our email fields (in the client, etc) as much as possible.